server/apps/distributed-app/docs/modules/ROOT/pages/configure/imap.adoc

```diff
@@ -47,11 +47,9 @@ Defaults to 0 (unlimited). Must be a positive integer, optionally with a unit: B
 | Deprecated. Should use `auth.plainAuthEnabled`, `auth.requireSSL` instead.
 Whether to enable Authentication PLAIN if the connection is not encrypted via SSL or STARTTLS. Defaults to `true`.
 
-| auth.plainAuthEnabled
-| true or false. Defaults to `true`.
-
 | auth.requireSSL
-| true or false. Defaults to `true`.
+| true or false. Defaults to `true`. This is a new way to configure Authentication PLAIN. Similar to plainAuthDisallowed.
+Whether to enable Authentication PLAIN if the connection is not encrypted via SSL or STARTTLS.
```

Comment:
We don't care about the authentication type here: all type of authentications should be rejected.